.nx/cache/lockfile.hash

```diff
@@ -0,0 +1 @@
+14961770178601404707
```

Comment:
Are `.nx/` files something that should be committed?

Reply:
@spmonahan I didn't even see them in the git log...should this be done in gitignore ideally?

Reply:
Following up with the build team. I've seen this in a few PRs recently and it _looks_ like something that probably shouldn't be comitted but I'm not sure.

Reply:
Based on the `.gitignore` in the repo root `.nx/cache` should be ignored: https://github.com/microsoft/fluentui/blob/master/.gitignore#L161

And given that the [cache only lives for a week](https://nx.dev/recipes/running-tasks/change-cache-location) it certainly seems like something that should be ignored.

Reply:
wc-3 didnt get all the latest changes from our master config and tooling. we don't have enough resources to support feature branches in monorepo and long term we dont wanna leverage this approach for maintenance reasons. 

wondering what's the story behind merging wc3 back to master ?

Reply:
Levi started and is working through conflicts. Either way, resolved with this PR.